fastlane/Fastfile

```diff
@@ -0,0 +1,179 @@
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+  previous_version_number = (sh "awk -F\\\" \'/\"version\":\/ {print $4}\' package.json").strip
+  UI.message("ℹ️  Current version: #{previous_version_number}")
+  UI.message("ℹ️  Setting version: #{new_version_number}")
+  files_to_update = [
+    'package.json',
+    'ios/RNPurchases.m',
+    'android/build.gradle',
+  ]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    for file_to_update in files_to_update
+      increment_build_number(previous_version_number, new_version_number, file_to_update)
+    end 
+  end
+end
+
+desc "Build example"
+lane :build_example do |options|
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+
+  sh("yarn example")
+  Dir.chdir("examples/purchaseTester")
+  sh("npx pod-install")
+end
+
+desc "Prepare for release by bumping verion and building the example."
+lane :prepare_for_release do |options|
+  bump_version(options)
+  build_example(options)
+end
+
+desc "Update hybrid common pod and gradle"
+lane :update_hybrid_common do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("RNPurchases.podspec")
+    Dir.chdir("..")
+  end  
+  previous_version_number=parse_previous_hybrid_version
+  # Add to options so if we're chaining lanes together, the other lanes get it
+  options[:previous_hybrid_version] = previous_version_number
+
+  UI.message("ℹ️  Current Purchases version: #{previous_version_number}")
+  UI.message("ℹ️  Setting Purchases version: #{new_version_number}")
+  files_to_update = [
+    'RNPurchases.podspec',
+    'android/build.gradle',
+  ]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    for file_to_update in files_to_update
+      increment_build_number(previous_version_number, new_version_number, file_to_update)
+    end 
+  end
+end
+
+desc "Build hybrid example"
+lane :build_hybrid_example do |options|
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+
+  sh("yarn example")
+  Dir.chdir("examples/purchaseTester/ios") do
+    sh("pod update PurchasesHybridCommon")
+  end
+  options[:previous_pod_version] = parse_pod_version
+end
+
+desc "Update iOS pod version during hybrid update"
+lane :update_ios_pod do |options|
+  if !File.file?("RNPurchases.podspec")
```

Comment:
if this is only meant to be called when updating something else and not independently, perhaps it's better to have it as a private method instead of a lane?